src/utils.js

```diff
@@ -1,3 +1,21 @@
+/** Transform a number between 0 and 1 into a pourcentage number (0.8 => 80%)
+ * @param { Number{0..1} } number
+ * @return { String }
+*/
 export function formatPercentage(number) {
   return Math.round(number * 100 * 100) / 100 + " %";
 }
+
+/** Transform Object into Array
+ * @param { Object } obj
+ * @return { Array }
+*/
+export function hydratation(obj) {
+  const hydratedData = [];
+
+  for (const [key, value] of Object.entries(obj)) {
+    hydratedData.push(value)
+  }
+
+  return hydratedData;
+}
```

Comment:
@Thomasevano @Pierre-Alain8  what do you think ? Is it not overkill ?

Reply:
De mon point de vue non car je l'utilise a plusieurs endroits et cela permet de fluidifier la lecture plutot que d'avoir une loop dans une loop ect

Reply:
Si c'est utiliser plusieurs fois oui, par contre je comprends pas le nom, "hydratation" ?

Reply:
+ Cela évite de répéter cette action d’hydratation dans  chaque script où il est nécessaire d'utiliser cette méthode si j'ai bien compris ? @LimongiVirgil

Reply:
Thomas => Hydratation c'est le terme dev pour dire que tu transforme une structure de data en une autre si je dit pas de connerie

PA => oui c'est ça

Reply:
https://tech-fr.netlify.app/articles/fr515100/index.html @Thomasevano 
Sinon ok pour moi la PR @LimongiVirgil